test/clj/puppetlabs/trapperkeeper/services/webserver/jetty9_service_handlers_test.clj

```diff
@@ -301,4 +343,4 @@
                                                                 :follow-redirects false})]
           (is (= (:status response) 302))
           (is (= (get-in response [:headers "location"]) "http://localhost:8080/hello/"))
-          (is (= (get-in response [:opts :url]) "http://localhost:8080/hello")))))))
\ No newline at end of file
+          (is (= (get-in response [:opts :url]) "http://localhost:8080/hello")))))))
```

Comment:
Ah, sorry about that. I think we should definitely have a trailing newline at the end of files, so no changes necessary here